shared_memory/main.cpp

```diff
@@ -300,7 +306,17 @@ int main(int argc, char **argv)
 		}
 	}
 
-	if (load_all || load_bd) {
+	if(load_all || load_faction_assoc) {
```

Comment:
Small:

`if(load_all` vs `if (load_all`

Reply:
yeah, I just copied load_bd branch, which doesn't match with the rest of the code :P